academy/manager.py

```diff
@@ -55,14 +62,6 @@ class _RunSpec(Generic[AgentT, ExchangeTransportT]):
 async def _run_agent_on_worker_async(
```

Comment:
^ this is now more like run agent in event loop, rather than "on worker", naming-wise